mitthesis.cls

```diff
@@ -180,10 +180,10 @@
     \typeout{Single spaced}
     \def\mystretch{1}}	
 
-%% `vi' and `upcase' document style options.  Krishna Sethuraman (1990)
-\newcount\vithesis
-\DeclareOption{vi}{\typeout{Course VI/VIII thesis style.}\advance\vithesis by1}
-\vithesis=0
+%% Pre-2023 theses are copyrighted to MIT instead of the author.
+\newcount\oldthesis
```

Comment:
should we name this something more descriptive, like `mitowned`? It is maybe conceivable that someone might have MIT own their thesis in the future, or that MIT will change their policies again.